public/config.default.json

```diff
@@ -1,5 +1,5 @@
 {
   "apiUrl": "http://localhost:3002",
-  "paymailDomain": "bux.com",
+  "paymailDomain": "example.com",
   "wsUrl": "ws://localhost:8080/api/websocket"
```

Comment:
Sorry for this 🙇 
but can you set the same port as in apiUrl, so the port 3002 🙏

Reply:
Do you mean - for `wsUrl`?

Reply:
yes in wsUrl
```suggestion
  "wsUrl": "ws://localhost:3002/api/websocket"
```